modules/couchbase-cluster/variables.tf

```diff
@@ -42,72 +49,84 @@ variable "user_data" {
 # ---------------------------------------------------------------------------------------------------------------------
 
 variable "ssh_key_name" {
-  description = "The name of an EC2 Key Pair that can be used to SSH to the EC2 Instances in this cluster. Set to an empty string to not associate a Key Pair."
-  default     = ""
+  description = "The name of an EC2 Key Pair that can be used to SSH to the EC2 Instances in this cluster. Set to null to not associate a Key Pair."
+  type        = string
+  default     = null
 }
 
 variable "allowed_ssh_cidr_blocks" {
   description = "A list of CIDR-formatted IP address ranges from which the EC2 Instances will allow SSH connections"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
 variable "allowed_ssh_security_group_ids" {
   description = "A list of security group IDs from which the EC2 Instances will allow SSH connections"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
 variable "termination_policies" {
   description = "A list of policies to decide how the instances in the auto scale group should be terminated. The allowed values are OldestInstance, NewestInstance, OldestLaunchConfiguration, ClosestToNextInstanceHour, Default."
```

Comment:
```suggestion
  description = "The policy to decide how the instances in the auto scale group should be terminated. The allowed values are OldestInstance, NewestInstance, OldestLaunchConfiguration, ClosestToNextInstanceHour, Default."
```

Reply:
The original description is deceiving.

Reply:
I think it should actually be a list: https://github.com/gruntwork-io/terraform-aws-couchbase/pull/49/commits/31d9a70aaba2d4546225efd1c0c023e1931f0304